msmbuilder/msm/src/transmat_mle_prinz.c

```diff
@@ -4,6 +4,8 @@
 #include <stdlib.h>
 #include "transmat_mle_prinz.h"
 
+#define MAX_ITER 10000
```

Comment:
Is that enough iterations to fit most matrices? I don't have an intuition about what the order of magnitude for the iteration count should be.